src/transformers/models/vit_msn/modeling_vit_msn.py

```diff
@@ -632,6 +632,8 @@ def forward(
         >>> from PIL import Image
         >>> import requests
 
+        >>> torch.manual_seed(2)
```

Comment:
Since the classification head of ViT MSN is randomly initialized we need this for a consistent output (`print(model.config.id2label[predicted_label])`).